compiler/rustc_query_system/src/query/mod.rs

```diff
@@ -26,27 +27,29 @@ use thin_vec::ThinVec;
 ///
 /// This is mostly used in case of cycles for error reporting.
 #[derive(Clone, Debug)]
-pub struct QueryStackFrame {
-    pub name: &'static str,
+pub struct QueryStackFrame<D: DepKind> {
+    pub name: D,
```

Comment:
is intentional to have both `name` and `dep_kind` below? they're always going to be the same, no?

Reply:
`name` can be removed yeah, probably an accident

Reply:
Fixed